### ISSUE TITLE
"Lookahead exceeds token buffer length" with simple grammar

### ISSUE THREAD

**Issue by Leandros** [jsinger67/parol#54]
Trying to parse anything with a parser generated from this elementary grammar:

```
%start Foo
%title "Foo grammar"
%comment "Empty grammar generated by `parol`"
%line_comment "//"

%%

Rjs: 'Hello' 'world';
```

Trying to parse 

```
Hello world
```

results in the following errors:

```
  × Failed parsing file ./test.txt
  ├─▶ Failed accessing lookahead token!
  ╰─▶ Lookahead exceeds token buffer length
```

The error can be fixed by adding the empty case to the production:

```
Rjs: 'Hello' 'world' | ;
```

Is this intended behavior?

**Comment by jsinger67** [comment]
@Leandros Thank you for reporting this.
I know about this behavior but it is not good documented and actually easy to fix.
The cause of this is that your first grammar ia actually k0. This results in a lookahead buffer of size 0.
I think I will fix this simply by setting the lower size limit of the lookahead buffer to 1.

**Comment by jsinger67** [comment]
Fixed with commit c302ba2.

**Comment by Leandros** [comment]
Wonderful. Thank you!
